src/libraries/Directory.Build.targets

```diff
@@ -265,7 +265,7 @@
 
   <!-- The Default behavior in VS is to show files for the first target framework in TargetFrameworks property.
         This is required to show all the files corresponding to all target frameworks in VS. -->
-  <ItemGroup>
+  <ItemGroup Condition="'$(SkipNoneItemGlobbing)' != 'true'">
```

Comment:
Where did this cause a problem?

Reply:
Because of this code path, all items under the project were globbed (all source files under src/libraries). I noticed that None items were included in the binlog and opening them took VERY long.

Reply:
Ouch. We have had that bug multiple times.  Make sure we still get right behavior in VS. perhaps we should even condition this to only apply in VS.